gems/pending/disk/modules/MSCommon.rb

```diff
@@ -349,4 +363,5 @@ def self.checksum(buf, skip_offset)
     # GRRRRR - convert to actual 32-bits.
     [~csum].pack('L').unpack('L')[0]
   end
+  private_class_method :checksum
```

Comment:
@jerryk55 can you review these changes.  Are these private methods in this file?  The gist is that class/singleton methods that follow the private keyword aren't actually private.  You can't make them private in that way.  You need to use our preferred way, `private_class_method`, or make them private in the singleton class.